pkgs/tools/security/step-ca/default.nix

```diff
@@ -1,19 +1,30 @@
-{ lib, buildGoPackage, fetchFromGitHub }:
+{ lib, stdenv, pkgconfig, pcsclite, buildGoModule, fetchFromGitHub, darwin }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "step-ca";
-  version = "0.13.3";
+  version = "0.14.6";
 
   goPackagePath = "github.com/smallstep/certificates";
 
   src = fetchFromGitHub {
     owner = "smallstep";
     repo = "certificates";
     rev = "v${version}";
-    sha256 = "1i42j7v5a5qqqb9ng8irblfyzykhyws0394q3zac290ymjijxbnq";
+    sha256 = "0v8dxn5zbvnpjl9qx68igmy5515b67l3k684v8qw9vqzzx4z570c";
   };
 
-  goDeps = ./deps.nix;
+  vendorSha256 = "042wksjxjjxzyh2vi4yxfdds8yrmv82i1fll056m8w5dchd0bskb";
+
+  nativeBuildInputs = [ pkgconfig ];
+
+  buildInputs =
+    lib.optional stdenv.isLinux (lib.getDev pcsclite)
+    ++ lib.optional stdenv.isDarwin (darwin.apple_sdk.frameworks.PCSC);
+
+  # pull in go-piv/piv-go#75
+  # once go-piv/piv-go#75 is merged and released, we should
+  # use the released version (and push upstream to do the same)
+  patches = [ ../yubikey-agent/use-piv-go-75.patch ];
```

Comment:
Getting this from another package is dirty, and considering the version this patch is pointing to breaks darwin (see https://github.com/go-piv/piv-go/pull/75), and `piv-go` might not be needed at all, let's reach out to upstream on their `go.sum` file.

Reply:
`piv-go` _is_ needed. Package doesn't build without it.

Reply:
There are two packages being bumped here.